### DISCUSSION TITLE
Worked perfectly.

### DISCUSSION THREAD

**Discussion by Fcrate** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#191] [General]
Thank you, you're a life saver. Worked perfectly on Poco F3. Exactly as instructed.

**Comment by Darpan3011** [comment]
> Thank you, you're a life saver. Worked perfectly on Poco F3. Exactly as instructed.

**Reply by Darpan3011** [reply]
Hello brother. Can you provide me the file folder that I have to use. I am not able to understand the steps that he provided. Thanks!

**Reply by Fcrate** [reply]
Did you download php 8.0?
Download it, and all the files in his post, download notepad++, then use it to make the edits he mentioned.
Use "open with" in windows to run bypass.php

**Reply by Darpan3011** [reply]
Thanks for the reply brother!!
I am a software engineer so I know how to edit the file 😊 but I am stucking at the error "php error line 333 null pointer" while running bypass.php file

can you give me the folder structure in C drive?

1. We have to place this repo code in C drive. Also where do we have to install php ? in this repo code folder or anywhere else with some other folder name?
2. Also for internet connection on laptop have you used the mobile hotspot for which you were trying to unlock bootloader or some other device for laptop's internet connection.
